diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index 00ae69d..f68a8f7 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -102,6 +102,7 @@ exports.networks = [
     { chainId: 1285n, shortName: 'mriver' },
     { chainId: 1287n, shortName: 'mbase' },
     { chainId: 1294n, shortName: 'bobabeam' },
+    { chainId: 1315n, shortName: 'story-aeneid' },
     { chainId: 1329n, shortName: 'sei' },
     { chainId: 1337n, shortName: 'geth' },
     { chainId: 1442n, shortName: 'testnet-zkEVM-mango' },
@@ -116,6 +117,8 @@ exports.networks = [
     { chainId: 1984n, shortName: 'euntest' },
     { chainId: 1998n, shortName: 'kyoto-testnet' },
     { chainId: 2000n, shortName: 'dc' },
+    { chainId: 1514n, shortName: 'story' },
+    { chainId: 1516n, shortName: 'story-odyssey' },
     { chainId: 2001n, shortName: 'milkada' },
     { chainId: 2002n, shortName: 'milkalgo' },
     { chainId: 2008n, shortName: 'cloudwalk_testnet' },
